cards.c

```diff
@@ -0,0 +1,56 @@
+/* Write a program cards.c that implements a deck of cards. Implement the suits as an enum, and the cards as a struct containing the value and suit. Implement a function drawCard() to generate a random card and getSuit() to print out the the suit properly. In main, draw and print out 5 cards in the following format:
```

Comment:
You were close - you forgot to include a suit and a couple of values, which might be why the format of your output wasn't always quite right. Sometimes it only gave 4 cards, other times it only printed "7 of " when it probably should have been hearts. Also, comments. They're incredibly important.